.gitignore

```diff
@@ -65,4 +65,7 @@ docs/site
 outputs.json
 
 # go packages
-test/pkg
\ No newline at end of file
+test/pkg
+
+# File created by tagging resource
+modules/AWS/asg_ec2_tagging
```

Comment:
aws tagging creates such a file, and it'd be nice to have it in gitignore to avoid committing it unintentionally.

Reply:
Do we still need tagging module while using launch template tags for eks?